lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -779,6 +787,14 @@ def GetMigrationStatus(self):
 
     return self.Execute("query-migrate")
 
+  @_ensure_connection
+  def GetVersion(self):
```

Comment:
perhaps this extra function `GetVersion()` can be avoided? a version information is already available from the QMP [`connect()`/`greeting`](https://github.com/ganeti/ganeti/blob/3368a7b6a7dc5e038caa68ebaa2ef96eab65efdb/lib/hypervisor/hv_kvm/monitor.py#L296). And used in [`SetMigrationParameters()`](https://github.com/ganeti/ganeti/blob/3368a7b6a7dc5e038caa68ebaa2ef96eab65efdb/lib/hypervisor/hv_kvm/monitor.py#L716)

Reply:
Yes, I entirely overlooked that the version is already parsed from the QMP greeting/handshake. I reverted to using the `version` property.